salt/states/nexus.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+'''
+This state downloads artifacts from Nexus 3.x.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def downloaded(name, artifact, target_dir='/tmp', target_file=None):
```

Comment:
This state should have a `__virtual__()` function to gate loading it as well.

Reply:
Added __virtual__ function